### PR TITLE
Combine InternalService with ManagementIService

### PR DESCRIPTION
**Pull request by JackieTien97**
1. delete mpp_port config property, only use internal_port.
2. delete ManagementIService and move its functions into InternalService

### CODE REVIEW

**Comment by MrQuansy** [comment]
LGTM

File: server/src/main/java/org/apache/iotdb/db/service/thrift/impl/InternalServiceImpl.java [hunk 1]
```diff
@@ -25,23 +25,29 @@
...
 import org.apache.iotdb.db.exception.DataRegionException;
 import org.apache.iotdb.db.exception.metadata.IllegalPathException;
 import org.apache.iotdb.db.exception.metadata.MetadataException;
 import org.apache.iotdb.db.metadata.path.PartialPath;
 import org.apache.iotdb.db.metadata.schemaregion.SchemaEngine;
+import org.apache.iotdb.db.mpp.common.FragmentInstanceId;
+import org.apache.iotdb.db.mpp.execution.FragmentInstanceInfo;
+import org.apache.iotdb.db.mpp.execution.FragmentInstanceManager;
+import org.apache.iotdb.db.mpp.sql.analyze.QueryType;
+import org.apache.iotdb.mpp.rpc.thrift.*;
```
Comment (CRZbulabula) [inline server/src/main/java/org/apache/iotdb/db/service/thrift/impl/InternalServiceImpl.java:47]: Why import *?

File: thrift/src/main/thrift/mpp.thrift [hunk 2]
```diff
@@ -16,9 +16,31 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+include "common.thrift"
 namespace java org.apache.iotdb.mpp.rpc.thrift
 
+struct CreateSchemaRegionReq {
```
Comment (CRZbulabula) [inline thrift/src/main/thrift/mpp.thrift:22]: Rename CreateSchemaRegionReq to TCreateSchemaRegionReq might be better.

File: thrift/src/main/thrift/mpp.thrift [hunk 3]
```diff
@@ -16,9 +16,31 @@
...
+    1: required common.TRegionReplicaSet regionReplicaSet
+    2: required string storageGroup
+}
+
+struct CreateDataRegionReq {
```
Comment (CRZbulabula) [inline thrift/src/main/thrift/mpp.thrift:27]: Rename CreateDataRegionReq to TCreateDataRegionReq might be better.

**Review by CRZbulabula — COMMENTED** [review]
PTAL~

**Review by CRZbulabula — APPROVED** [review]
LGTM~

**Pull request merged**